{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 1}, "tests": {"0": {"status": "ERROR", "message": "request = <SubRequest 'setup' for <Function test_e2e>>\n\n    @pytest.fixture(scope=\"class\")\n    \n    def setup(request):\n        #service_obj = Service()\n        #service_obj = Service(\"C:/Users/HP/PycharmProjects/chromedriver.exe\")\n        #driver = webdriver.Chrome(service=service_obj)\n>       driver = webdriver.Chrome(executable_path=\"C://Users/HP/PycharmProjects/chromedriver.exe\")\nE       TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nconftest.py:17: TypeError\n", "test_name": "test_e2e", "rerun": "0"}}, "suite_name": "test_E2E.py"}}}, "date": "February 19, 2024", "start_time": 1708337712.803502, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "0", "skip": "0", "error": "1", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}